README.md

```diff
@@ -33,7 +33,7 @@ We are not currently accepting public contributions - see our [contributions](ht
 * [Known issues](https://docs.improbable.io/unreal/latest/known-issues)
 
 ## Unreal Engine changes
-In order to transform Unreal from a single server engine to a distributed model, we had to make a small number of changes to UE4 code. We will attempt to consolidate and remove (or submit as PR to Epic) as many of these changes as possible. You can see the changes in our forked [Unreal Engine repo, `4.20-SpatialOSUnrealGDK` branch](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK).
+In order to transform Unreal from a single server engine to a distributed model, we had to make a small number of changes to UE4 code. We will attempt to consolidate and remove (or submit as PR to Epic) as many of these changes as possible. You can see the changes in our forked [Unreal Engine repo, `4.20-SpatialOSUnrealGDK-release` branch](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK-release).
```

Comment:
Slightly late to this - but the github link should probably be - https://github.com/improbableio/UnrealEngine
We've setup the appropriate default branch on github.